build/webpack.config.js

```diff
@@ -190,8 +190,30 @@ const mapJsChunkConfig = (buildMode) => {
   }
 }
 
+const copyPluginConfig = () => {
+  return {
+    entry: [
+      path.join(__dirname, '..', 'node_modules', '@mapbox', 'mapbox-gl-rtl-text', 'mapbox-gl-rtl-text.js.min')
```

Comment:
i think this is the same as       path.join('@mapbox', 'mapbox-gl-rtl-text', 'mapbox-gl-rtl-text.js.min')